depresolve/depresolve_test.go

```diff
@@ -0,0 +1,44 @@
+// +build nettest
+
+package depresolve_test
+
+import (
+	"reflect"
+	"testing"
+
+	"sourcegraph.com/sourcegraph/srclib-go/depresolve"
+	"sourcegraph.com/sourcegraph/srclib/dep"
+)
+
+// TestResolveImportPath tests the behavior of ResolveImportPath
+// when called on some common Go package import paths.
+func TestResolveImportPath(t *testing.T) {
+	tests := []struct {
+		ImportPath string
+		Result     *dep.ResolvedTarget
+	}{
+		{"k8s.io/kubernetes/pkg/api", &dep.ResolvedTarget{"https://github.com/kubernetes/kubernetes", "github.com/kubernetes/kubernetes/pkg/api", "GoPackage", "", ""}},
+		{"gopkg.in/inconshreveable/log15.v2", &dep.ResolvedTarget{"https://gopkg.in/inconshreveable/log15.v2", "gopkg.in/inconshreveable/log15.v2", "GoPackage", "", ""}},
+		{"azul3d.org/semver.v2", &dep.ResolvedTarget{"https://azul3d.org/semver.v2", "azul3d.org/semver.v2", "GoPackage", "", ""}},
+		{"sourcegraph.com/sourcegraph/srclib/graph", &dep.ResolvedTarget{"https://sourcegraph.com/sourcegraph/srclib", "sourcegraph.com/sourcegraph/srclib/graph", "GoPackage", "", ""}},
+		{"sourcegraph.com/sourcegraph/sourcegraph/app", &dep.ResolvedTarget{"https://sourcegraph.com/sourcegraph/sourcegraph", "sourcegraph.com/sourcegraph/sourcegraph/app", "GoPackage", "", ""}},
+		{"google.golang.org/grpc", &dep.ResolvedTarget{"https://github.com/grpc/grpc-go", "github.com/grpc/grpc-go", "GoPackage", "", ""}},
```

Comment:
I think you should also add `google.golang.org/grpc/codes` test case, since that's the interesting one and the one that was failing before you fixed the bugs.

It'd be good to the same for the special-cased cloud and api repos too.